### ISSUE TITLE
--clearKeychain breaks keychain until you reboot the simulator

### ISSUE THREAD

**Issue by santiagofm** [wix/AppleSimulatorUtils#62]
**Description**
Upon upgrading to Xcode 11.1 we started seeing an error when trying to access the Keychain from e2e on our app, the native API throws the following error:
```swift 
errSecNotAvailable = -25291 /* No Keychain is available. */
```

After some investigation we found that the issue arises after clearing the keychain using applesimutils:
```js
await device.deviceDriver.applesimutils._execAppleSimUtils({
  args: `--simulator "${device._deviceId}" --clearKeychain`,
});
```

We need to reboot the simulator for it to work again.

**Steps to Reproduce**
https://github.com/santiagofm/detoxKeychainIssue

Taking a look at the directory referenced in [ClearKeychain.m](https://github.com/wix/AppleSimulatorUtils/blob/90855d16d969b2c1a3a0bba6107b0f34ae5419b1/applesimutils/applesimutils/ClearKeychain.m), we see all files being removed but **not** fully re-created until we reboot the simulator.

**Environment**
 - macOS version: 10.14.6
 - Xcode version: 11.X
 - Detox: 14.4.1
 - applesimutils: 0.7.2

**Comment by LeoNatan** [comment]
I kill the securityd daemon, delete and start it again, which should take care of itself. I will take a look when I have time. In the meantime, you can add `--restartSB` to reboot the simulator.

**Comment by santiagofm** [comment]
@LeoNatan That did not work.
Seems like a Xcode 11.1 issue because building & running with Xcode 11 works fine!

**Comment by santiagofm** [comment]
@LeoNatan Why not `xcrun simctl shutdown` & ``xcrun simctl boot` instead of messing with the securityd daemon?

**Comment by LeoNatan** [comment]
Because this is much faster, if it works. There is no messing; `launchctl` is responsible for daemons and its API is public and well documented. I need to check why the daemon is not recreating its files when launching. Maybe the daemon changed its name in macOS/iOS 13.

**Comment by LeoNatan** [comment]
Fixed in 0.7.3
